builder/azure/common/client/config.go

```diff
@@ -209,12 +206,6 @@ func (c Config) Validate(errs *packersdk.MultiError) {
 		c.ClientSecret == "" &&
 		c.ClientCertPath == "" &&
 		c.ClientJWT != "" {
-		p := jwt.Parser{}
```

Comment:
Standard Claims are no longer supported, we could use map claims, but honestly this code feels extraneous